data/magl-laminas-legacy-wrapper.php

```diff
@@ -17,11 +17,19 @@
 }
 
 // Setup autoloading
-require 'init_autoloader.php';
+if (file_exists('init_autoloader.php')) {
+    require 'init_autoloader.php';
+} elseif (file_exists(__DIR__ . '/../vendor/autoload.php')) {
+    include __DIR__ . '/../vendor/autoload.php';
+}
```

Comment:
Since this file is only an example, IMHO we can force that the module is executed within a composer environment.
```suggestion
if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
    include __DIR__ . '/../vendor/autoload.php';
} else {
    die('please use composer to install all dependencies.');
}
```